internal/handlers/tigris/tigrisdb/stats.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tigrisdb
+
+import (
+	"context"
+
+	"github.com/tigrisdata/tigris-client-go/driver"
+
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+)
+
+// CollectionStats describes statistics for a Tigris collection.
+type CollectionStats struct {
+	NumObjects int32
+	Size       int64
+}
+
+// FetchStats returns a set of statistics for the given database and collection.
+func (tdb *TigrisDB) FetchStats(ctx context.Context, param FetchParam) (*CollectionStats, error) {
```

Comment:
Let's move it to function to make it easier to switch to using transactions / "querier" later

Reply:
@AlekSi do you mean such a function should have something like `FetchStats(ctx context.Context, db driver.Database, param FetchParam)` as a signature?

(See an example here: https://github.com/FerretDB/tigris-experiments/pull/5/files#diff-63ab601287b8b9c040760fe0bdd288f55b73f37cd7e4f1e519bea2bd43a18bbaR59)

Reply:
Yeah

Reply:
(we could also switch to passing FetchParam by a pointer)

Reply:
Fixed